store/src/java-test/com/zimbra/cs/filter/NotifyTest.java

```diff
@@ -112,4 +112,37 @@ public void filterValidToField() {
 
     }
 
+    @Test
+    public void testNotifyMailtoWithHeaderVariable() {
+        String sampleMsg = "from: abc@zimbra.com\n"
+                + "Subject: Hello\n"
+                + "to: test@zimbra.com\n";
+        String filterScript = "require [\"enotify\", \"variables\"];\n"
+            + "set \"to\" \"nick\";\n"
+            + "if anyof (header :contains [\"Subject\"] \"Hello\") {\n"
+            + "notify \"test2@zimbra.com\" \"\" \"${SUBJECT} ${to}\"\n"
```

Comment:
Please test the notify action (legacy version) using the body-template with the template-specific variables, such as ${SUBJECT}, ${TO}, ${CC}, and ${BODY} (see the docs/soap.txt for more details).  Those are predefined variables for the legacy notify action.